setup.php

```diff
@@ -69,6 +69,9 @@
 	{$DVWAUploadsWrite}<br />
 	{$DVWAPHPWrite}<br />
 	<br />
+	<br />

```

Comment:
Why are these two <br /> here? Would expect the three lines to be kept together as they are all relevant to file/directory permissions.